packages/react-sdk/generator/index.js

```diff
@@ -26,6 +28,10 @@ const isPrimitiveType = (type) => {
 const isViemType = (type) => {
   return ["Hex", "Address"].includes(type);
 };
+
+const isEnclosedInCurlyBraces = (type) => {
```

Comment:
```
const isEnclosedInCurlyBraces = (type) => {
  return type.startsWith("{") && type.endsWith("}");
};

```

slice(0, 1) and slice(-1) are replaced by startsWith("{") and endsWith("}"), which are more explicit and improve readability.